setupServers.js

```diff
@@ -0,0 +1,157 @@
+    );
+};
+
+export function setupHTTP(
+    {sequelizeManager, serverMessageReceive, mainWindow, OPTIONS}
+    ) {
+
+    const server = restify.createServer();
+    server.use(restify.queryParser());
+    server.use(restify.CORS({
+        origins: ['*'],
+        credentials: false,
+        headers: ['Access-Control-Allow-Origin']
+    })).listen(OPTIONS.port);
+
+    setupRoutes(server, serverMessageReceive(
+        sequelizeManager, mainWindow.webContents)
+    );
+}
+
+export function setupHTTPS(
+    {sequelizeManager, serverMessageReceive, mainWindow, OPTIONS}
+) {
+
+    const keyFile = `${__dirname}/ssl/certs/server/privkey.pem`;
+    const csrFile = `${__dirname}/ssl/certs/server/fullchain.pem`;
+
+    const hosts = '/etc/hosts';
+    const connectorURL = 'connector.plot.ly';
+
+    const options = {
+        name: 'Connector',
+        process: {
+            options: {
+                env: {'VAR': 'VALUE'}
+            },
+            on: (ps) => {
+                ps.stdout.on('data', function(data) {console.log(data);});
+                setTimeout(function() {
+                    ps.kill();
+                }, 50000);
+            }
+        }
+    };
+
+    const scriptsDirectory = replace(/\ /g, '\\ ', `${__dirname}`);
+
+    // redirect connectorURL to localhost
+    try {
+        fs.readFile(hosts, function (err, data) {
+            if (data.indexOf(connectorURL) < 0) {
+                showSudoMessage();
+                sudo.exec(`sh  "${scriptsDirectory}"/ssl/redirectConnector.sh`,
+                    options, function(error) {
+                    if (error) {
+                        dialog.showMessageBox(
+                            {type: 'info', buttons: ['OK'], message: 'error'}
+                        );
+                        console.log(error);
+                    } else {
+                        console.log(`${connectorURL} is now wired to local.`);
+                    }
+                });
+            } else {
+                console.log(`${connectorURL} is already wired to a local port.`);
+            }
+        });
+    } catch (error) {
+        console.log(error);
+    }
+
+    // setup HTTPS server with self signed certs
+    try {
+        // try reading certs
+        fs.accessSync(keyFile, fs.F_OK);
+        fs.accessSync(csrFile, fs.F_OK);
+
+        console.log('Found certs files.');
+        setupSecureRestifyServer(
+            {keyFile, csrFile, sequelizeManager,
+            serverMessageReceive, mainWindow, OPTIONS}
+        );
+    } catch (e) {
+        console.log('Did not find certs, generating...');
+        // if error returned, certs do not exist -- let's create them
+        showSudoMessage();
+        sudo.exec(`sh  "${scriptsDirectory}"/ssl/createKeys.sh`, options, function(error) {
+            if (error) {
+                dialog.showMessageBox(
+                    {type: 'info', buttons: ['OK'], message: error.toString()}
+                );
```

Comment:
Let's change this message to be like:

> Yikes, an error occurred while setting up HTTPS. Create an issue at https://github.com/plotly/plotly-database-connector/issues and paste the following error message:
> ${error.toString()}

Also, can we use the `logger` here?